deployment/main.tf

```diff
@@ -0,0 +1,116 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "aws_ecs_cluster" "this" {
+  name = "akka-app-cluster"
+  tags = {
+    Project = "aipcc-akka"
+  }
+}
+
+resource "aws_iam_role" "ecs_task_execution" {
+  name = "ecsTaskExecutionRole"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [{
+      Effect = "Allow",
+      Principal = {
+        Service = "ecs-tasks.amazonaws.com"
+      },
+      Action = "sts:AssumeRole"
+    }]
+  })
+  tags = {
+    Project = "aipcc-akka"
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "ecs_task_execution_attach" {
+  role       = aws_iam_role.ecs_task_execution.name
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
+}
+
+resource "aws_security_group" "akka_sg" {
+  name        = "akka-internal-sg"
+  description = "Allow internal access to ECS task on port 5000"
+  vpc_id      = "vpc-0ff68e46a55526721"
+
+  ingress {
+    from_port   = 5000
+    to_port     = 5000
+    protocol    = "tcp"
+    cidr_blocks = ["10.0.0.0/8"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+  tags = {
+    Project = "aipcc-akka"
+  }
+}
+
+resource "aws_ecs_task_definition" "akka_app" {
+  family                   = "akka-app-task"
+  requires_compatibilities = ["FARGATE"]
+  network_mode             = "awsvpc"
+  cpu                      = "512"
+  memory                   = "1024"
+  execution_role_arn       = aws_iam_role.ecs_task_execution.arn
+
+  container_definitions = jsonencode([
+    {
+      name      = "akka-app"
+      image     = "images.paas.redhat.com/rhel-ai-cicd/akka:latest"
```

Comment:
It's best practice to avoid the latest tag and use a specific image version to prevent unexpected changes.

Reply:
We currently just have `latest` tag

Reply:
Yup. Tha was a general comment to address in the future as soon as we apply release tags.